rskj-core/src/main/java/co/rsk/util/HexUtils.java

```diff
@@ -74,6 +74,20 @@ public static BigInteger stringHexToBigInteger(final String input) {
         return new BigInteger(hexa, 16);
     }
 
+    /**
+     * Convert hex encoded string, even if it doesn't have the prefix 0x, to decoded BigInteger
+     */
+    public static BigInteger forceParseStringHexToBigInteger(final String input) {
+        String inputWithPrefix = input;
+
+        if(!hasHexPrefix(input)) {
+            inputWithPrefix = "0x" + input;
+        }
+
+        String hexa = inputWithPrefix.substring(2);
+        return new BigInteger(hexa, 16);
```

Comment:
We are not handling this possible `NumberFormatException` in the caller method, is it possible?